ExampleCheckout/UITests/PaymentSession/Model/Customer.swift

```diff
@@ -8,10 +8,10 @@ import Foundation
 
 struct Customer: Codable {
     /// Customer identifier given by the merchant. Not validated for uniqueness by OPG.
-    var number: String
+    let number: String
 
     /// Customer e-mail address. It is highly recommended to provide it, since it is a mandatory information for some PSP and often used by advanced risk management.
-    var email: String?
+    let email: String
```

Comment:
Should be marked as optional to keep consistency with [a server model](https://www.optile.io/reference#operation/createPaymentSession).

```suggestion
    let email: String?
```